docs/reference/src/language/syntax/properties.md

```diff
@@ -150,3 +150,69 @@ export component Example inherits Window {
     }
 }
 ```
+
+<!--
```

Comment:
I guess this won't show up just yet, which makes sense ... until we make it stable?

Reply:
Yes that's the idea